_examples/crds/couchbase.tf

```diff
@@ -12,7 +12,7 @@ metadata:
   name: name-here-cluster
 spec:
   baseImage: name-here-image
-  version: name-here-image-version
+  version: 6.0.1
```

Comment:
this was needed cause your regex from the `^([\w\d]+-)?\d+\.\d+.\d+(-[\w\d]+)?$` crd didn't allow the original version